_releases/ocp-4.16-test-jobs-arm64.json

```diff
@@ -1,9 +1,4 @@
 {
-  "nightly": [
```

Comment:
@wangke19 suggest to submit a PR to disable prow job as well. if you won't add informing job for arm64

Reply:
For tight resources of x86_64, I choose to keep the job of arm64.

Reply:
ok, so you will add another job in new PR for arm64?

Reply:
AWS DR jobs of amd64 is different from arm64, including FIPS. so it is better to disable prow job for arm64